lib/php/rendition-factory/src/Transformer/Document/PdfToImageTransformerModule.php

```diff
@@ -39,7 +39,24 @@ public function buildConfiguration(NodeBuilder $builder): void
         // @formatter:off
         $builder
             ->arrayNode('options')
-            ->ignoreExtraKeys(false)
+                ->children()
+                    ->scalarNode('extension')
```

Comment:
can be `stringNode`

Reply:
The stringNode() method was introduced in Symfony 7.2. we are in sf 6.4
https://symfony.com/doc/current/components/config/definition.html#variable-nodes

Reply:
right !